runtime/ts/storage/firebase-storage.ts

```diff
@@ -1342,10 +1367,14 @@ class FirebaseBigCollection extends FirebaseStorageProvider {
   // Returns a FirebaseCursor id for paginated reads of the current version of this BigCollection.
   // The id should be passed to cursorNext() to retrive the contained entities. The cursor itself
   // is held internally by this collection so we can discard it once the stream read has completed.
-  async stream(pageSize) {
+  //
+  // By default items are returned in order of original insertion into the collection (with the
+  // caveat that items removed during a streamed read may be returned at the end). Set forward to
+  // false to return items in reverse insertion order.
+  async stream(pageSize, {forward = true} = {}) {
```

Comment:
does it make sense to have both pageSize and forward as options set by the options object?

e.g. `async stream({pageSize = 10, forward = true} = {})`

Reply:
Yes, it probably should. I explicitly don't want to set a default for that, though.

Reply:
I might change that in a separate PR; it affects the in-memory version as well (and I have a couple of more CLs ready for this feature).